vello_shaders/src/compile/mod.rs

```diff
@@ -198,3 +204,11 @@ fn postprocess(wgsl: &str) -> String {
     }
     output
 }
+
+/// Returns the absolute path to the directory containing the WGSL shaders.
+///
+/// The path is determined at compile time.
```

Comment:
And may be invalid at runtime, especially if the program is deployed on a different machine

As a different point:
This is also a reproducible builds challenge, which I guess we should document?
Idk, maybe even making it `doc(hidden)`

Reply:
Sure, I added a note about the impact on reproducible builds.